packages/keystone/lib/ListTypes/utils.js

```diff
@@ -0,0 +1,95 @@
+const { upcase } = require('@keystonejs/utils');
+const { logger } = require('@keystonejs/logger');
+
+const keystoneLogger = logger('keystone');
+
+const preventInvalidUnderscorePrefix = str => str.replace(/^__/, '_');
```

Comment:
This is only used once and very simple - we could just inline it - but I won't hold this up for that